plugins/jobs/src/js/types/JobSchedule.ts

```diff
@@ -1,19 +1,35 @@
 import { Schedule as MetronomeSchedule } from "#SRC/js/events/MetronomeClient";
+import { ConcurrentPolicy } from "#PLUGINS/jobs/src/js/components/form/helpers/JobFormData";
+
 export interface JobSchedule {
   cron: string;
   enabled: boolean;
   id: string;
   startingDeadlineSeconds: number;
   timezone: string;
+  concurrencyPolicy: ConcurrentPolicy;
 }
 
+export type ConcurrentPolicyType = "ALLOW" | "FORBID";
+
+export const ConcurrentPolicySchema = `
```

Comment:
is there a difference between Concurren**t**Policy`and Concurren**cy**Policy? If not i'd  prefer to stick to one of those terms for better grep-/discoverability.

Reply:
ConcurrentPolicy is the name of the type and concurrencyPolicy is the name of the attribute.

Reply:
i just wanted to make sure that the difference in spelling was intentional. 👍